tests/w3c_tracecontext_validation_server.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
```

Comment:
That could be a good place, but I felt it wasn't accurate as currently the tracecontext test suite would fail without SDK behavior (specifically creating new spans when a spancontext is invalid from the propagator).